COLLABORATOR_GUIDE.md

```diff
@@ -294,22 +288,121 @@ After generating one, make sure to keep it safe because GitHub will not show the
+Then, it's mostly a standard npm package release process with running `npm version`, followed by an `npm publish`.
+
+```sh
+npm version {major|minor|patch}
+```
+
+Depending on the commits that have been merged, you can choose from `major`, `minor`, or `patch` as the versioning values.
+
+Afterwards, you want to push the commit and the tag to the repo.
+It's necessary to do this before running `npm publish` because our GitHub release automation
+relies on the commit being available on GitHub.
 
 ```sh
-VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=my-personal-access-token contrib release next patch
+git push origin master
 ```
 
-This makes a patch release, you can also do a `minor` and a `major` release.
+Finally, run `npm publish` with an appropriate tag. Don't forget to supply your token.
+
+```sh
+VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=my-personal-access-token npm publish --tag next
+```
 
 After it's done, verify that the GitHub release has the correct changelog output.
 
+If the GitHub release did not work correctly for whatever reason, you can run it manually:
+
+```sh
+VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=123 node build/gh-release.js --prelease
+```
+
+#### Video.js 5
+
+Make sure to go to the 5.x branch and grab the latest updates.
+
+```sh
+git checkout 5.x
+git pull origin 5.x
+```
+
+> *Note:* you probably need to delete v6 tags due to the way that the our CHANGELOG lib works.
+>
+> You can run this to delete them:
+> ```sh
+> git tag | grep '^v6' | xargs git tag -d
+> ```
+> This will find all tags that start with `^v6` and delete them.
+
+Then, we have a script that automates most of the steps for publishing. It's a little trickier than publishing v6.
+
+##### Edit git-semver-tags
+
+You'll need to edit `git-semver-tags` to support our usage of tags that are not part of the branch.
+In the file `node_modules/conventional-changelog-cli/node_modules/conventional-changelog/node_modules/conventional-changelog-core/node_modules/git-semver-tags/index.js`, edit the line that says sets the `cmd` to be:
+```js
+var cmd = 'git log --all --date-order --decorate --no-color';
+```
+
+#### And now for the release
+
+After getting rid of the tags and getting the latest updates, you can just run the release script:
+
+```sh
+VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=123 ./build/bin/release-next.sh
+```
+
+It will prompt you for a version change type, so, input `patch` or `minor` or `major`.
+
+When it's done building everything, it'll show you the commit that's made via the default pager (i.e., less).
+
+After exiting the pager, it'll make sure you want to continue with publishing.
+
+It will automatically release it as a `next-5` tag on npm.
+
+Then push the local changes up:
+
+```sh
+git push origin 5.x
+```
+
+Also, you'll need to copy the CHANGELOG for this version and manually edit the GitHub release to include it.
```

Comment:
Specifying where to copy the CHANGELOG from and where to put it(this could be an example) would be very useful.